tests/Tasks/UpdatePackageInfoTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace BringYourOwnIdeas\Maintenance\Tests\Tasks;
+
+use BringYourOwnIdeas\Maintenance\Util\ComposerLoader;
+use SapphireTest;
+use BringYourOwnIdeas\Maintenance\Tasks\UpdatePackageInfo;
+
+class UpdatePackageInfoTest extends SapphireTest
+{
+    public function testGetPackageInfo()
+    {
+        $loader = $this->createMock(ComposerLoader::class);
+        $loader->method('getLock')->willReturn(json_decode(<<<LOCK
```

Comment:
You may as well just define this as a PHP array and skip the decoding

Reply:
Yeah, I thought to do it that way - the major issue is that objects are actually objects (as opposed to keyed array elements). So far as I'm aware there's no shorthand for an inline object definition in PHP.

Reply:
You can typecast with `object` eg: https://3v4l.org/LppQB

Reply:
Yeah, I also considered that... but I then also though second of it in that it sort of results in obfuscating the mocked input. Leaving it as a readable JSON string I thought was more readable :)

Thoughts?